lib/foreman_maintain/upgrade_runner.rb

```diff
@@ -67,6 +67,9 @@ def run
           run_phase(phase)
         end
       end
+      unless quit?
```

Comment:
just a personal preference. Sometimes the modifier statement can get overlooked. Probably not this case though